### DISCUSSION TITLE
alternatives to System.Drawing

### DISCUSSION THREAD

**Discussion by charga** [picoe/Eto#2698] [Q&A]
I know tha eto can user GTK and SkiaSharp for drawing objects. Any consideration on another source? like Aspose.Drawing or IronDrawing, o MAUI

Regards, Carlos

**Comment by cwensley** [comment]
Eto only uses System.Drawing when targeting Windows Forms.  For WPF, macOS and GTK it doesn't use System.Drawing at all.

Using something other than System.Drawing when targeting Windows Forms wouldn't make any sort of sense as Windows Forms is based on it.
